app/screens/home/search/results/file_card.tsx

```diff
@@ -90,8 +93,21 @@ export default function FileCard({channelName, fileInfo}: Props) {
 
     // TODO: hook this up
     const handleThreeDotPress = () => {
-        /* eslint-disable no-console */
-        console.log('pressed 3 dot');
+        const renderContent = () => {
+            return (
+                <FileOptions
+                    fileInfo={fileInfo}
+                />
+
+            );
+        };
+        bottomSheet({
+            closeButtonId: 'close-search-file-options',
+            renderContent,
+            snapPoints: [400, 10],
```

Comment:
400 seems a magic number. How is it calculated? Should it be calculated in a more programatic way?

My point being: in a few months we decide to change the way we show the file Options. We make the changes there, and suddenly, the snapPoints are wrong. We have to remember come here to fix it.
If we use some kind of constant (probably defined in the FileOptions component) when we change that in the future, we can update the constant there and this should work "for free".

Reply:
Done.  This is now calculated depending on the HEADER_HEIGHT and the number of options shown.